patroni/__init__.py

```diff
@@ -48,13 +48,19 @@ def load_dynamic_configuration(self):
                 logger.warning('Can not get cluster from dcs')
 
     def get_tags(self):
-        return {tag: value for tag, value in self.config.get('tags', {}).items()
+        tags = {tag: value for tag, value in self.config.get('tags', {}).items()
                 if tag not in ('clonefrom', 'nofailover', 'noloadbalance') or value}
+        tags.update(self.ha.get_dynamic_tags())
```

Comment:
Sounds like a plan. I'll try to document this in a way that makes it clear why the tag can locally be False, but advertised as being True. I'm still kind of on the fence if conflating configured values and dynamic state in this way is a good idea. But I guess it isn't too hard to change in the future if it proves to be a bad idea.